reactive/etcd.py

```diff
@@ -129,6 +131,14 @@ def install_etcd():
             pkg_list = ['etcd']
             apt_update()
             apt_install(pkg_list, fatal=True)
+            # Stop the service and remove the defaults
+            # I hate that I have to do this. Sorry short-lived local data #RIP
+            # State control is to prevent upgrade-charm from nuking cluster
+            # data.
+            if not is_state('etcd.package.adjusted'):
+                host.service('stop', 'etcd')
+                rmtree('/var/lib/etcd/default')
+                set_state('etcd.package.adjusted')
             set_state('etcd.installed')
             return
```

Comment:
Don't you need to start the etcd service before returning from the "install_etcd" method here? The service was running, I don't know what would start it again.